extensions/widgets/trending/src/components/cards/latest-topics.tsx

```diff
@@ -34,6 +38,15 @@ export type LatestTopicsProps = {
   refetchTagSubscriptions: () => void;
 };
 
+/**
+ * This utility method compares two arrays of tags and returns the difference,
+ * @param arr1 - first array to be compared
+ * @param arr2 - second array to be compared
+ * @returns an array of strings
+ */
+const getTagsDiff = (arr1: string[], arr2: string[]) =>
```

Comment:
what if the lists are sorted different? this doesn't look it does what it is expected